packages/components/src/components/hds/app-header/index.ts

```diff
@@ -50,7 +50,7 @@ export default class HdsAppHeader extends Component<HdsAppHeaderSignature> {
   private _menuContentId = 'hds-menu-content-' + guidFor(this);
 
   // we use the `lg` breakpoint for `desktop` viewports, but consumers can override its value
```

Comment:
probably should update the comment too?

Reply:
🤦 Yep, I should! Thanks for catching that!